apps/web/hooks/useDownloadResults.tsx

```diff
@@ -0,0 +1,59 @@
+import getConfig from 'next/config';
+import axios from 'axios';
+import {useEffect, useState} from 'react';
+import {useRouter} from 'next/router';
+import {useSession} from 'next-auth/react';
+
+const useDownloadResults = () => {
+  const {publicRuntimeConfig} = getConfig();
+  axios.defaults.baseURL = publicRuntimeConfig.REACT_APP_API_URL || "http://localhost:3333/api"
+
+  const [response, setResponse] = useState(null);
+  const [error, setError] = useState(null);
+  const [loading, setLoading] = useState(false);
+
+  const router = useRouter();
+
+  const {data} = useSession()
+
+  useEffect(() => {
+    if (!data) {
+      router.push('/');
+    }
+  }, [data])
+
+  const getDownloadResultsOperation = async (trialId, nct_id) => {
+    setLoading(true);
+    const accessToken = localStorage.getItem('ctims-accessToken');
+    const headers = {
+      'Authorization': 'Bearer ' + accessToken,
+    }
+
+    try {
+      const csvBlob = await axios.request({
+        method: 'post',
+        url: `/trial-result/` + trialId + '/export',
```

Comment:
looks like you are using backticks for sting interpolation but then you are concatenating the string